modules/backup/main.tf

```diff
@@ -1,14 +1,35 @@
 locals {
   cold_storage_after = 30
+  is_production      = can(regex("production", terraform.workspace))
```

Comment:
This is quite neat! What made you decide on this instead of using the standard approach of...
`is-production    = substr(terraform.workspace, length(local.application_name), length(terraform.workspace)) == "-production"`

Reply:
local.application_name does not exist in this module, I did initially copy and paste that line, but went with this as it works and doesn't require anymore locals.